Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -839,7 +849,29 @@ public async Task Delete( TItem item )
     /// <returns>A task that represents the asynchronous operation.</returns>
     public async Task Save()

```

Comment:
It does, it's a bit more clear. 

But I was just leaving the original implementation where the save/validation concern was handled to the edit components. I.E the code thats executed in **Save** is as close as possible to the code executed by using the **save click** (**_SaveWithValidation_** ) events.

 (In case new logic shows up in the **_SaveWithValidation_** method and gets lost if you're not explicitly calling it) --> My only real concern.

But I think the **UseValidation** check is indeed better to have even if the end result is the same.

Reply:
I think both Modal and EditRow just need to execute `parentDataGrid.Save`. It is little clearer to me that way.

Reply:
If they both call `parentDataGrid.Save` then my concern goes away. I don't think you've made that change though did you?

Reply:
> I don't think you've made that change though did you?

I didn't. Wanted to discuss it with you first.

Reply:
I'm up for it. As long as everything calls the same thing. I think it's good.

Reply:
Going ahead and going to apply the suggestion